### DISCUSSION TITLE
Please tell me your favorite Manga (comic).

### DISCUSSION THREAD

**Discussion by Fta98** [NAIST-SE/AttractingContributors#841] [2023 Attraction Contributions]
What is your favorite Manga (comic)?
Please tell me your best one!
Here is my repo: https://github.com/Fta98/AttractingContributors-Manga/blob/main/README.md

**Comment by oto10** [comment]
I added my favorite manga to your list.
Could you please add your favorite music to my README.md?
https://github.com/oto10/AttractingContributors-your-favorite-music
Thank you in advance.

**Comment by TakumaMiwa** [comment]
I added One punch man to your list! please help me!
https://github.com/TakumaMiwa/AttractingContributors-Music
